src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -123,125 +165,109 @@ public inherited sharing class RD2_OpportunityEvaluationService {
 
     /**
      * @description Primary processor
-     * @return RD2_OpportunityEvaluationService The service instance
+     * @return RD2_OpportunityEvaluationService This service instance
      */
     public RD2_OpportunityEvaluationService evaluateOpportunities() {
+        //do not evaluate Opps when override is set
+        if (isOverride) {
+            return this;
+        }
+        
+        RD2_ScheduleService scheduleService = new RD2_ScheduleService();
+        List<npe03__Recurring_Donation__c> newOppRDs = new List<npe03__Recurring_Donation__c>();
+        List<npe03__Recurring_Donation__c> updateOppRDs = new List<npe03__Recurring_Donation__c>();
+        List<npe03__Recurring_Donation__c> closedRDs = new List<npe03__Recurring_Donation__c>();
+
+        for (npe03__Recurring_Donation__c rd : rds) {  
+            rd.npe03__Next_Payment_Date__c = 
```

Comment:
@force2b , the opp service uses the correct Next Donation Date when creating/updating open Opps, but it does not updated the RD.  A new WI has been created for this, but the change can be made in this PR if required.

Reply:
The change has been made to update RD Next Donation Date from the Opp evaluation service.